### PR TITLE
Fix upstream search not work with gzip

### PR DESCRIPTION
**Pull request by Meeeeow**
Also fixed a small bug.

### CODE REVIEW

File: lib/up-storage.js [hunk 1]
```diff
@@ -353,17 +354,28 @@ Storage.prototype.search = function(startkey, options) {
...
 
-    res.pipe(JSONStream.parse('*')).on('data', (pkg) => {
-      if (Utils.is_object(pkg)) {
-        stream.emit('data', pkg);
-      }
-    });
-    res.on('end', () => {
+
+    // See https://github.com/request/request#requestoptions-callback
+    // Request library will not decode gzip stream.
```
Comment (Meeeeow) [inline lib/up-storage.js]: @juanpicado Use registry.yarnpkg.com as upstream, then search with npm cli

Comment (juanpicado) [inline lib/up-storage.js]: Thanks, I'll try out

Comment (Meeeeow) [inline lib/up-storage.js]: You really reply me at 22 hours ago? i have receive notification email at 1 hours ago

Comment (juanpicado) [inline lib/up-storage.js]: I did not press the "submit review" 😅😅😅😅 .. doh ... 😊 my bad. I'll need some days to review this. I'm 🚅ling these days.

Comment (juanpicado) [inline lib/up-storage.js]: @Meeeeow I messed up your PR :(

Comment (Meeeeow) [inline lib/up-storage.js]: I will resolve conflicts later, busy with work...

**Comment by Meeeeow** [comment]
Done @juanpicado

**Comment by juanpicado** [comment]
Thanks dude I'll release a new version soon

**Pull request merged**